data/campaigns/The_Deceivers_Gambit/units/Orcish_Adept.cfg

```diff
@@ -21,7 +21,7 @@
     advances_to=Orcish Shaman
     cost=21
     usage=archer
-    description= _ "Female orcs with spiritual leanings may find themselves inducted into the arts of orcish sorcery. Initially called adepts, these newly-trained orcs gain a curious influence over the shamanistic forces that separate life from non-life. Directed inwards, this art gives them great resilience against most forms of magic. Directed outwards, it grants them the fearsome power to siphon life from their enemies.
+    description= _ "Female orcs with spiritual leanings may find themselves inducted into the arts of orcish sorcery. Initially called Adepts, these newly-trained orcs gain a curious influence over the shamanistic forces that separate life from non-life. Directed inwards, this art gives them great resilience against most forms of magic. Directed outwards, it grants them the fearsome power to siphon life from their enemies.
 
 Although physically frail compared to other orcs this shamanistic art makes even a freshly trained adept a force to be reckoned with on the battlefield."
```

Comment:
I take the adept here to be the same Orcish Adept referenced in the previous line... although it seems strange it is in the entry for Shaman.